src/EA.Weee.RequestHandlers.Tests.Unit/Admin/AddAatfRequestHandlerTests.cs

```diff
@@ -78,7 +78,7 @@ public async Task HandleAsync_ValidInput_AddsAatf()
         {
             IWeeeAuthorization authorization = A.Fake<IWeeeAuthorization>();
 
-            AatfData aatf = new AatfData(Guid.NewGuid(), "name", "approval number", A.Dummy<Core.Shared.UKCompetentAuthorityData>(), Core.AatfReturn.AatfStatus.Approved, A.Dummy<AatfAddressData>(), Core.AatfReturn.AatfSize.Large, DateTime.Now);
+            AatfData aatf = new AatfData(Guid.NewGuid(), "name", "approval number", (Int16)2019, A.Dummy<Core.Shared.UKCompetentAuthorityData>(), Core.AatfReturn.AatfStatus.Approved, A.Dummy<AatfAddressData>(), Core.AatfReturn.AatfSize.Large, DateTime.Now);
```

Comment:
Do a check for the compliance year in the must have happened part below

Reply:
Added